### PR TITLE
docs(ci): add trigger update to docs.pact.io

### PR DESCRIPTION
**Pull request by YOU54F**
sync job not currently present in docs.pact.io repo on master, but will be later today

### CODE REVIEW

File: .github/workflows/trigger_pact_docs_update.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - '**.md'
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger docs.pact.io update workflow
+        run: |
+          curl -X POST https://api.github.com/repos/pact-foundation/docs.pact.io/dispatches \
+                -H 'Accept: application/vnd.github.everest-preview+json' \
```
Comment (tienvx) [inline .github/workflows/trigger_pact_docs_update.yml:18]: I search for the keyword "application/vnd.github.everest-preview+json" and I think this is quite old. Do we aware about this? https://developer.github.com/changes/3/

Maybe this is new syntax https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event

File: .github/workflows/trigger_pact_docs_update.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                -H "Authorization: Bearer $GITHUB_TOKEN" \
+                -d '{"event_type": "pact-php-docs-updated"}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GHTOKENFORTRIGGERINGPACTDOCSUPDATE }}
```
Comment (tienvx) [inline .github/workflows/trigger_pact_docs_update.yml:22]: It's hard to read. If we are already using it for several projects I fine with it.

**Review by tienvx — APPROVED** [review]
Approved. There are some comments but if we are already using this code several places it's fine to me.

File: .github/workflows/trigger_pact_docs_update.yml (hunk 2 above)
Comment (YOU54F) [reply .github/workflows/trigger_pact_docs_update.yml:22]: it's set at an organisation level, but agree it is hard to read.

If we update the webhook, I'm happy to change this to be easier to grok (as we will have to update each repos workflow anyway)

**Pull request merged**